tools/MVP/template.py

```diff
@@ -117,6 +133,9 @@ def _generate_files(name: str, language: str, include_setup: bool, output_direct
         help="Whether to include setup files such as a launch script (and CMakeLists.txt for C++).",
     )
     parser.add_argument("-o", "--output-dir", required=True, help="The absolute path to output the generated files to.")
+    parser.add_argument(
+        "-t", "--include-tests", action=BooleanOptionalAction, help="Whether to include basic test files for the presenters and models"
+    )
     args = parser.parse_args()
 
-    _generate_files(args.name[:1].upper() + args.name[1:], args.language, args.include_setup, args.output_dir)
+    _generate_files(args.name[:1].upper() + args.name[1:], args.language, args.include_setup, args.output_dir, args.add_tests)
```

Comment:
I think this needs to be `args.include_tests` rather than `args.add_tests` for the parser to recognise the argument

Reply:
I missed these, sorry.